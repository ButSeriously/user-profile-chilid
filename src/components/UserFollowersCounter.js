import PropTypes from 'prop-types';
import React from 'react';

const UserFollowersCounter = props => (
  <div className="user_followers__box">
    <p className="user_followers__values">
      { props.userStats[1] }
    </p>
    <p className="user_followers__names">
      { props.userStats[0] }
    </p>
  </div>
);

UserFollowersCounter.propTypes = {
  userStats: PropTypes.array.isRequired,
};

export default UserFollowersCounter;
