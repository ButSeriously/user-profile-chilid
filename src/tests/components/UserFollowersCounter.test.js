import React from 'react';
import { shallow } from 'enzyme';
import UserFollowersCounter from '../../components/UserFollowersCounter';

test('should render UserFollowersCounter', () => {
  const userStats = ['Stat', '25'];
  const wrapper = shallow(<UserFollowersCounter userStats={userStats} />);
  expect(wrapper).toMatchSnapshot();
});
