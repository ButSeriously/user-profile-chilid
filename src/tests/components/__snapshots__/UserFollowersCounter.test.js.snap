// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UserFollowersCounter 1`] = `
<div
  className="user_followers__box"
>
  <p
    className="user_followers__values"
  >
    25
  </p>
  <p
    className="user_followers__names"
  >
    Stat
  </p>
</div>
`;
